0.4.7/handlers/phpvms5/flights/search.php

```diff
@@ -144,7 +144,15 @@
     $query .= ' AND enabled=1';
 }
 
-$query .= ' ORDER BY id DESC LIMIT 100';
+$limit = 100;
+if ($_GET['limit'] !== null) {
+    assertData($_GET, array('limit' => 'int'));
+    $limit = intval($_GET['limit']);
+    $limit = $limit > 100 ? 100 : $limit;
+}
+
+$query .= ' ORDER BY submitdate DESC LIMIT ' . $limit;
+
```

Comment:
Does this mean we are adding pagination to flight searching? If so, where is this done?

Reply:
We are not adding pagination, the behavior remains exactly the same, we can just change the limit (but the limit will never exceed 100)